examples/package-lock.json

```diff
@@ -0,0 +1,104 @@
+{
```

Comment:
We probably shouldn't commit this file - like with the integration tests, we always want the latest deps, not what was being used the last time the examples were updated.